scripts/performance/summarize

```diff
@@ -12,14 +12,45 @@ Run this script with::
 
 And that should output::
 
-    Total time: 1.810 seconds
-    Max memory: 114.0 MiB
-    Max cpu: 208.3 percent
-    Average memory: 67.3 MiB
-    Average cpu: 140.5 percent
+    +-----------------------+----------+----------------------+
+    | Metric                | Mean     | Standard Deviation   |
```

Comment:
It would be nice somewhere in the output to mention how many files are being summarized.